packages/edit-site/src/components/add-new-template/new-template.js

```diff
@@ -263,6 +249,24 @@ function useMissingTemplates(
 	setEntityForSuggestions,
 	setShowCustomTemplateModal
 ) {
+	const DEFAULT_TEMPLATE_SLUGS = applyFilters(
```

Comment:
I'm generally wary of implementing JS filters because of the potential performance hit, but I don't _think_ this is a hot path so it should be okay.

A question for other reviewers is I'm wondering if it'd be better to get these slugs from the server and filter them server side?